packages/ui/src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -174,15 +174,13 @@ class Container extends React.Component<Props, ComponentState> {
                         ? NONE_RESOLVER_KEY
                         : nextProps.entity.resolverType
 
-                    const isMultiValueDisabled = this.isMultiValueDisabled(nextProps.entity, this.props.actions)
```

Comment:
Think you need more complex logic here other than simply removing this function.

Changing to be non multivalue would have still allowed conditions that are invalid to be exist which might could cause errors.  Do you remove all those conditions on actions?  Seems easier to block but I guess you can ask @swadheen

Reply:
I put it back.